wasmcontext.go

```diff
@@ -34,72 +34,74 @@ func (i *Instance) compile(wasmbytes []byte) {
 	// XQD Stubbing -{{{
 	// TODO: All of these XQD methods are stubbed. As they are implemented, they'll be removed from
 	// here and explicitly linked in the section below.
-	linker.DefineFunc("env", "xqd_log_endpoint_get", i.wasm3("xqd_log_endpoint_get"))
```

Comment:
This is all derived from: https://docs.rs/fastly-sys/0.4.0/fastly_sys/index.html